helsingborg-stad/app-mitt-helsingborg
# Refactor/assets images

## Explain the changes you’ve made
Refactor the usage of assets like icons, illustrations and slides.
Fixed typings and import of assets, using import instead of require.

## Explain why these changes are made
Many of the assets are either duplicates or not used at all, hence removing those. 

## How to test

1. Checkout this branch
2. Fire upp the simulator by running the command `yarn ios` or `yarn android`
3. Run the application as normal, all icons and illustrations should be the same
4. Add a banner image in form-builder and check if its working in a form

## Tested environments

- [] Storybook on a iOS device/simulator.
- [] Building the Application on a Android device/simulator.
